docker/launch-netbox.sh

```diff
@@ -1,8 +1,33 @@
 #!/bin/bash
 
-UNIT_CONFIG="${UNIT_CONFIG-/etc/unit/nginx-unit.json}"
 UNIT_SOCKET="/opt/unit/unit.sock"
 
+put_config() {
+  RET=$(
+    curl \
+    --silent \
+    --write-out '%{http_code}' \
+    --request PUT \
+    --data-binary "@$1" \
+    --unix-socket $UNIT_SOCKET \
+    http://localhost/$2
```

Comment:
I think I'd change the order of the arguments, so that the 'more' static parts are first. What I mean:

```suggestion
    --data-binary "@${2}" \
    --unix-socket $UNIT_SOCKET \
    "http://localhost/${1}"
```

The invocation would then be `put_config "config" "file.json"` and `put_config "certificate/bla" "bla.pem"`.
But I don't see any objective argument for one or the other, so if you like your argument order that's good with me.